From 70dbd785a505cd2945c702b2dda8315b40a9ac97 Mon Sep 17 00:00:00 2001
From: Jacopo Mondi <jacopo.mondi@ideasonboard.com>
Date: Thu, 29 Feb 2024 09:23:09 +0100
Subject: [PATCH 0918/1085] media: ov64a40: Name the subdev state variables to
 'state'

The subdev state arguments passed to the v4l2 subdev operations are
named 'sd_state' or 'state'.

Increase the driver consistency and use 'state' wherever possible.

Signed-off-by: Jacopo Mondi <jacopo.mondi@ideasonboard.com>
---
 drivers/media/i2c/ov64a40.c | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

--- a/drivers/media/i2c/ov64a40.c
+++ b/drivers/media/i2c/ov64a40.c
@@ -3072,7 +3072,7 @@ static int ov64a40_init_cfg(struct v4l2_
 }
 
 static int ov64a40_enum_mbus_code(struct v4l2_subdev *sd,
-				  struct v4l2_subdev_state *sd_state,
+				  struct v4l2_subdev_state *state,
 				  struct v4l2_subdev_mbus_code_enum *code)
 {
 	struct ov64a40 *ov64a40 = sd_to_ov64a40(sd);
@@ -3086,7 +3086,7 @@ static int ov64a40_enum_mbus_code(struct
 }
 
 static int ov64a40_enum_frame_size(struct v4l2_subdev *sd,
-				   struct v4l2_subdev_state *sd_state,
+				   struct v4l2_subdev_state *state,
 				   struct v4l2_subdev_frame_size_enum *fse)
 {
 	struct ov64a40 *ov64a40 = sd_to_ov64a40(sd);
@@ -3110,12 +3110,12 @@ static int ov64a40_enum_frame_size(struc
 }
 
 static int ov64a40_get_selection(struct v4l2_subdev *sd,
-				 struct v4l2_subdev_state *sd_state,
+				 struct v4l2_subdev_state *state,
 				 struct v4l2_subdev_selection *sel)
 {
 	switch (sel->target) {
 	case V4L2_SEL_TGT_CROP:
-		sel->r = *v4l2_subdev_get_pad_crop(sd, sd_state, 0);
+		sel->r = *v4l2_subdev_get_pad_crop(sd, state, 0);
 
 		return 0;
 
@@ -3141,7 +3141,7 @@ static int ov64a40_get_selection(struct
 }
 
 static int ov64a40_set_format(struct v4l2_subdev *sd,
-			      struct v4l2_subdev_state *sd_state,
+			      struct v4l2_subdev_state *state,
 			      struct v4l2_subdev_format *fmt)
 {
 	struct ov64a40 *ov64a40 = sd_to_ov64a40(sd);
@@ -3155,7 +3155,7 @@ static int ov64a40_set_format(struct v4l
 
 	ov64a40_update_pad_fmt(ov64a40, mode, &fmt->format);
 
-	format = v4l2_subdev_get_pad_format(sd, sd_state, 0);
+	format = v4l2_subdev_get_pad_format(sd, state, 0);
 	if (ov64a40->mode == mode && format->code == fmt->format.code)
 		return 0;
 
@@ -3166,7 +3166,7 @@ static int ov64a40_set_format(struct v4l
 		int exp_max;
 
 		ov64a40->mode = mode;
-		*v4l2_subdev_get_pad_crop(sd, sd_state, 0) = mode->analogue_crop;
+		*v4l2_subdev_get_pad_crop(sd, state, 0) = mode->analogue_crop;
 
 		/* Update control limits according to the new mode. */
 		timings = ov64a40_get_timings(ov64a40,
